packages/markdown/components/Image/index.jsx

```diff
@@ -15,7 +15,7 @@ class Image extends React.Component {
     const { align, title, alt, width, height, className = '' } = this.props;
     const extras = { align, width, height };
     return (
-      <React.Fragment>
+      <div>
         <img
```

Comment:
I think making such heavy use of the fragment component may also have been a contributing factor to this perf bottleneck. Definitely heavier than a native `<div/>`.